<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditor</class>
 <widget class="QWidget" name="DialogEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButtonOk">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonInsertChild">
     <property name="text">
      <string> \-&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonRemove">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <column>
      <property name="text">
       <string>Tree</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
